### ISSUE TITLE
UAT | EFiling | Address field mismatch

### ISSUE THREAD

**Issue by anshumanth-egov** [pucardotorg/dristi#535]
Address fields while adding complainant details are different from fields filled during registration. As we are doing auto-population, you might want to fix this if we want to incorporate updating backend with details entered during case filing
![Uploading Screenshot 2024-06-20 at 1.10.36 PM.png…]()

**Comment by anirudh-0** [comment]
@anshumanth-egov can you post a screenshot again? The one attached doesn't seem to work

**Comment by anshumanth-egov** [comment]
![image](https://github.com/pucardotorg/dristi/assets/173333707/52e5d4d0-e622-4b7d-8844-ebb5aec65b16)

![image](https://github.com/pucardotorg/dristi/assets/173333707/9b339372-5aeb-4a48-a3e5-5f137c85701e)

**Comment by anirudh-0** [comment]
@anshumanth-egov this is a design level change, can you come back to us on what is the right set of fields to use ?

**Comment by anshumanth-egov** [comment]
I don't think we have specified it in the PRD anywhere. I think either format is fine, my concern is just that since the data for both can come from same table, is there a potential issue we might run into?
Maybe @atulgupta2024 can comment.

**Comment by rajeshcherukumalli** [comment]
Hi @Susmitabe , for Address fields the changes are implemented with other field , please check and confirm the same.
